core/tx_pool.go

```diff
@@ -1677,6 +1681,22 @@ func (pool *TxPool) demoteUnexecutables() {
 	}
 }
 
+// PauseReorgs stops any new reorg jobs to be started but doesn't interrupt any existing ones that are in flight
```

Comment:
Would be good to mention in the comment that these methods are blocking. For now it's not an issue but might lead to issues/unexpected behavior if multiple callers use them.